Tests/Serializer/PlainTextDeserializationVisitorFunctionalTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Smartbox\CoreBundle\Tests\Serializer;
+
+use JMS\Serializer\Naming\IdenticalPropertyNamingStrategy;
+use JMS\Serializer\SerializerBuilder;
+use Smartbox\CoreBundle\Serializer\PlainTextDeserializationVisitor;
+
+class PlainTextDeserializationVisitorFunctionalTest extends \PHPUnit\Framework\TestCase
+{
+    public function testSerialization()
```

Comment:
I would call the method `testDeserialization()` because is what ia the purpose of the test (try to deserialize a string)

Reply:
Renamed and added a PHPDOC as well